src/components/Book.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+
+function Book({ book }) {
+  return (
+    <>
+      <div className="bookCard">
+        <div className="bookInfo">
+          <p className="genre">{book.genre}</p>
+          <p className="title">{book.title}</p>
+          <p className="author">{book.author}</p>
+          <ul>
+            <li><button type="button">Comments</button></li>
+            <li><button type="button">Remove</button></li>
+            <li><button type="button">Edit</button></li>
+          </ul>
+        </div>
+        <div className="progress">
+          <div>
+            <div>chart</div>
+            <p>percentage</p>
+          </div>
+          <div>
+            <p>{book.status}</p>
+            <p>{book.currentChapter}</p>
+            <p>{book.chapterNumber}</p>
+            <button type="button">Update progress</button>
+          </div>
+        </div>
+      </div>
+      {' '}
+      <div className="bookCard">
+        <div className="bookInfo">
+          <p className="genre">{book.genre}</p>
+          <p className="title">{book.title}</p>
+          <p className="author">{book.author}</p>
+          <ul>
+            <li><button type="button">Comments</button></li>
+            <li><button type="button">Remove</button></li>
+            <li><button type="button">Edit</button></li>
+          </ul>
+        </div>
+        <div className="progress">
+          <div>
+            <div>chart</div>
+            <p>percentage</p>
+          </div>
+          <div>
+            <p>{book.status}</p>
+            <p>{book.currentChapter}</p>
+            <p>{book.chapterNumber}</p>
+            <button type="button">Update progress</button>
+          </div>
+        </div>
+      </div>
+      <div className="bookInfo">
+        <p className="genre">{book.genre}</p>
+        <p className="title">{book.title}</p>
+        <p className="author">{book.author}</p>
+        <ul>
+          <li><button type="button">Comments</button></li>
+          <li><button type="button">Remove</button></li>
+          <li><button type="button">Edit</button></li>
+        </ul>
+      </div>
+      <div className="progress">
+        <div>
+          <div>chart</div>
+          <p>percentage</p>
+        </div>
+        <div>
+          <p>{book.status}</p>
+          <p>{book.currentChapter}</p>
+          <p>{book.chapterNumber}</p>
+          <button type="button">Update progress</button>
+        </div>
+      </div>
```

Comment:
- [ ][Optional] this code is not actually needed since you are already looping through the books array in `Books.js`, kindly remove

Reply:
thanks, let me do it.